content/docs/command-reference/plots/index.md

```diff
@@ -260,3 +277,48 @@ file:///Users/usr/src/plots/classes.csv.html
 
 > A confusion matrix [template](/doc/command-reference/plots#plot-templates) is
 > predefined in DVC (found in `.dvc/plots/confusion.json`).
+
+### Example for the HTML template
+
+Download the Vega libraries into the directory where you'll produce the
+`dvc plots`.
+
+```dvc
+$ wget https://cdn.jsdelivr.net/npm/vega@5.10.0 -O my_vega.js
+$ wget https://cdn.jsdelivr.net/npm/vega-lite@4.8.1 -O my_vega_lite.js
+$ wget https://cdn.jsdelivr.net/npm/vega-embed@6.5.1 -O my_vega_embed.js
+```
+
+Create the following HTML file with your text editor:
```

Comment:
Maybe specify the file name for the example here. It's a specific example anyway so it's fine.

Reply:
I mention it later, in sentence starting with "Supposing..."

Reply:
I'll make a specific suggestion about this part. Resolving